staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -61,6 +61,13 @@ const (
 	// storageWatchListPageSize is the cacher's request chunk size of
 	// initial and resync watch lists to storage.
 	storageWatchListPageSize = int64(10000)
+
+	// eventFreshDuration is time duration of events we want to keep.
```

Comment:
Looks like the comment was copied from previous code and needs to be updated. 

It describes internal motivation for value of `eventFreshDuration`. It needs to be enough to fit a bookmarking period. In the PR we are codifying a second mechanism, external one, we need to store events long enough for clients to open a watch after a list. Please document them both and  add validation to prevent setting it below bookmark frequency.

Reply:
I updated the const and the comment && also added appropriate comment above the relevant change in `pkg/controlplane/apiserver/options/options.go` explaining the context - I think this is more proper place for laying out this information.